lib/tasks/organisations.rake

```diff
@@ -3,9 +3,22 @@ namespace :organisations do
   task fetch: :environment do
     include AdvisoryLock::DSL
 
-    with_lock("forms-admin:organisations:fetch") do
-      OrganisationsFetcher.new.call
-    end
+    puts("Starting organisation fetch")
```

Comment:
I'm unsure whether there's a significant benefit to using `Rails.logger.info` rather than `puts`, but I think we've started using `Rails.logger.info` more in rake tasks. I'm not sure whether we actively decided this is the right thing to do though.
I think the main thing is just that if we use `puts` and don't add an assertion on the output in the tests, it will log to the console which is annoying - but that's not the case currently for the tests for this tasks so no major need to change.

Reply:
I'll swap it over, it does look more appropriate

Reply:
It's now using the logger, and the test has been updated to no longer worry about it